tests/Predis/Command/Redis/EXPIRE_Test.php

```diff
@@ -143,14 +143,14 @@ public function keysProvider(): array
                 ['noExpiry', 2, 'XX'],
             ],
             'only if new expiry is greater then current one' => [
-                ['newExpiryLower', 'value', 'EXAT', time() + 1000],
-                ['newExpiryGreater', 'value', 'EXAT', time() + 10],
-                ['newExpiryGreater', 20, 'GT'],
+                ['newExpiryLower', 'value', 'EX', 1000],
+                ['newExpiryGreater', 'value', 'EX', 10],
+                ['newExpiryGreater', 100, 'GT'],
                 ['newExpiryLower', 20, 'GT'],
             ],
             'only if new expiry is lower then current one' => [
-                ['newExpiryLower', 'value', 'EXAT', time() + 1000],
-                ['newExpiryGreater', 'value', 'EXAT', time() + 10],
+                ['newExpiryLower', 'value', 'EX', 1000],
+                ['newExpiryGreater', 'value', 'EX', 10],
```

Comment:
Because tests in one specific CI (PHP 8.1, Redis 7) breaks with EXAT param, can't reproduce it locally

Reply:
Why does it break on Redis 7 with 8.1 but not 8.0 or 8.2?

Reply:
I have no answer on this question and I suspect it will take a lot to understand, since it looks very specific and depends on CI environment, `EXAT` with value `time() + 1000`, have the same effect as `EX` with value `1000`, so I decide to use it instead